Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,24 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const durationInHours = tasks.map(task => task.duration / 60);
+  const filterHours = durationInHours.filter(hour => hour >= 2);
+  for (let i = 0; i < filterHours.length; i++) {
```

Comment:
Do you really need to use this for loop? Would perhaps chaining the `map` method work instead?

Reply:
Hi Hasan,
Thanks for taking the time to go through my homework. Changes have been made.